src/components/advancedResources/AdvancedResourcesCard.tsx

```diff
@@ -0,0 +1,12 @@
+import ResourcesCard from "../ResourcesCard";
+const AdvancedResourcesCard = ({ themes, filteredData }: any) => {
+  return (
```

Comment:
This component should be named `AdvancedResourcesSection`

Reply:
fixed